roadmap/implementors-guide/guide.md

```diff
@@ -1069,9 +1096,33 @@ Dispatch a `PovFetchSubsystemMessage(relay_parent, candidate_hash, sender)` and
 
 (TODO: send statements to Statement Distribution subsystem, handle shutdown signal from candidate backing subsystem)
 
+### Statement Distribution Subsystem
+
+The statement distribution subsystem sends statements to peer nodes, detects double-voting, and tabulates when a sufficient portion of the validator set has unanimously judged a candidate. When judgment is not unanimous, it escalates the issue to misbehavior arbitration.
+
+Statement Distribution is the only validity module subsystem which has any notion of peer nodes or of our own cryptographic keys. It is responsible for signing statements that we have generated and forwarding them, and for detecting a variety of peer node misbehaviors for reporting to Misbehavior Arbitration. Within the Validity module, it's the main point of contact (via the Overseer) with peer nodes. On receiving a signed statement from a peer, assuming the peer receipt state machine is in an appropriate state, it sends the Candidate Receipt to the Candidate Backing subsystem to double-check the peer's judgment.
+
+#### Peer Recipet State Machine
+
+There is a very simple state machine which governs which messages we are willing to receive from peer validators. Not depicted in the state machine: on initial receipt of any `SignedStatement`, validate that the provided signature does in fact sign the included data.
```

Comment:
I'd suggest shortening to peers as opposed to peer validators, as we don't know that any of our peers are validators and for the purposes of gossip, only assume that validators are transitively connected via a largely full-node graph.

Reply:
OK. I'd assumed that messages from non-validator peers just got dumped regardless as irrelevant; the whole point of separating out validators is that they're the ones who get to vote.

Reply:
Yeah, I see where you're coming from. Regular peers can't issue their own messages but they can safely relay messages signed by validators, so I think it still fits that worldview just with better propagation. And then we don't need to worry about peer authentication.